packages/runtime-node/src/worker-thread-entry.ts

```diff
@@ -53,7 +53,11 @@ const handleStartupMessage = async (command: WorkerThreadStartupCommand) => {
         childEnvName: environmentContextName,
     });
 
-    disposeNodeEnv = runningNodeEnv.dispose;
+    disposeNodeEnv = () => {
+        ensureWorkerThreadContext(parentPort);
```

Comment:
this was already validated during start. use `!` in next line instead?

Reply:
fixed